src/main/kotlin/de/triplet/gradle/play/AndroidPublisherHelper.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2014 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.triplet.gradle.play
+
+import com.google.api.client.auth.oauth2.Credential
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
+import com.google.api.services.androidpublisher.AndroidPublisher
+import com.google.api.services.androidpublisher.AndroidPublisherScopes
+import java.io.File
+
+class AndroidPublisherHelper {
+    companion object {
+        fun init(extension: PlayPublisherPluginExtension, config: PlayAccountConfig?): AndroidPublisher {
```

Comment:
Sure. Can be moved. The helper is using a `JacksonFactory` and a `HttpTransport` which should be mockable in the future